### PR TITLE
More commands

### PR DESCRIPTION
**Pull request by Mk-Chan**
I'm not exactly sure how to test `memory` and `cores`. Any suggestions?

### CODE REVIEW

**Comment by Mk-Chan** [comment]
On the other hand, it is just an API, it's really up to the engine so is there even a need to test such functions?

File: chess/xboard.py [hunk 1]
```diff
@@ -355,21 +355,42 @@ def hard(self, async_callback=None):
...
 
         return self._queue_command(command, async_callback)
 
+    def post(self, async_callback=None):
+        """
+        Command used to tell the engine to output it's analysis.
+
+        :return: Nothing
+        """
+        self.set_post(True)
```
Comment (niklasf) [inline chess/xboard.py]: This should probably be `return self.set_post(True)`, so that it can return a `Future` depending on `async_callback`.

Comment (Mk-Chan) [inline chess/xboard.py]: good catch, I think there's one more place i may have made the same mistake let me check

**Comment by niklasf** [comment]
Thank you, merged!

**Pull request merged**